Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1274,15 +1274,15 @@
     <!-- YTXT: Text for consent given -->
     <string name="submission_test_result_available_text_consent_given">"Vielen Dank, dass Sie mithelfen, andere zu warnen und damit einverstanden sind, Ihr Testergebnis zu teilen. \n\n"<b>"Bitte geben Sie Ihr Testergebnis im nächsten Schritt frei, indem Sie „Teilen“ antippen."</b>""</string>
     <!-- XHED: Close screen popup title for consent given -->
-    <string name="submission_test_result_available_close_dialog_title_consent_given">"Wollen Sie wirklich abbrechen?"</string>
+    <string name="submission_test_result_available_close_dialog_title_consent_given">"Achtung:\nSie sind dabei den Vorgang abzubrechen.\nSie haben Ihre Mitmenschen noch nicht gewarnt. Bitte schützen Sie Ihre Mitmenschen und schließen Sie den Vorgang ab."</string>
```

Comment:
This seems wrong. The title and body are switched now. Title is too long to be visible.
![device-2021-02-04-141213](https://user-images.githubusercontent.com/49635654/106897586-6a36e280-66f3-11eb-9cec-927331a56ca1.png)

Reply:
This ticket only solves the switched dialogs problem. It's known that the texts need to be changed and a new PR with the right string will be added when we get them

Reply:
I can see above and below in the strings file  that text has been switched. The popup before also shows that is was the other way around before.
![Uploading device-2021-02-08-114627.png…]()

Reply:
That's true, but check Luka's comment https://github.com/corona-warn-app/cwa-app-android/pull/2279#discussion_r569453901. They were wrongly named since the beginning.

Reply:
The content of submission_test_result_available_close_dialog_body_consent_given and submission_test_result_available_close_dialog_title_consent_given needs to be switched. They are not correct in the UI as shown above.

Reply:
Yes @chiljamgossow is right, the naming of the keys where wrong like `given` vs `not_given`. But now the body and the title are still interchanged.